### ISSUE TITLE
Unix Test Failures for v0.27.3

### ISSUE THREAD

**Issue by clanmills** [Exiv2/exiv2#1165]
The following tests are not run on Unix builds (NetBSD, SunOS, FreeBSD).  From test/Makefile:
```
TESTS2 = exifdata-test.sh     \
         exiv2-test.sh        \
         imagetest.sh         \
         iotest.sh            \
         iptctest.sh          \
         preview-test.sh      \
         tiff-test.sh         \
         write-test.sh        \
         xmpparser-test.sh 
```
I'll investigate an make an assessment.

| Test | Comment |
|:-- |:-- |
| exifdata-test.sh     | |
| exiv2-test.sh | |
| imagetest.sh | |
| iotest.sh | |
| iptctest.sh | |
| preview-test.sh | |
| tiff-test.sh | |
| write-test.sh | |
| xmpparser-test.sh | |

**Comment by clanmills** [comment]
These issues are all due to the same single cause - binary data in the test output.

**Comment by clanmills** [comment]
Fixed submitted #1207.
